jobs/e2e-runner.groovy

```diff
@@ -0,0 +1,127 @@
+evaluate(new File("${WORKSPACE}/common.groovy"))
+
+import utilities.StatusUpdater
+
+[ [type: 'master', branch: 'master'],
+  [type: 'pr', branch: '${sha1}'],
+].each { Map config ->
+  isMaster = config.type == 'master'
+  isPR = config.type == 'pr'
+
+  name = isMaster ? "e2e-runner" : "e2e-runner-${config.type}"
+  downstreamJobName = defaults.testJob[config.type]
+
+  job(name) {
+    description """
+      Watches the workflow-e2e repo for a ${config.type} commit
+    """.stripIndent().trim()
+
+    scm {
+      git {
+        remote {
+          github("deis/e2e-runner")
+          credentials('597819a0-b0b9-4974-a79b-3a5c2322606d')
+          if (isPR) {
+            refspec('+refs/pull/*:refs/remotes/origin/pr/*')
+          }
+        }
+        branch(config.branch)
+      }
+    }
+
+    if (isPR) {
+      concurrentBuild()
+      throttleConcurrentBuilds {
+        maxPerNode(defaults.maxBuildsPerNode)
+        maxTotal(defaults.maxTotalConcurrentBuilds)
+      }
+    }
+
+    logRotator {
+      daysToKeep defaults.daysToKeep
+    }
+
+    if (isPR) { // set up GitHubPullRequest build trigger
+      triggers {
+        pullRequest {
+          admin('deis-admin')
+          cron('H/5 * * * *')
+          useGitHubHooks()
+          triggerPhrase('OK to test')
+          orgWhitelist(['deis'])
+          allowMembersOfWhitelistedOrgsAsAdmin()
+          // this plugin will update PR status no matter what,
+          // so until we fix this, here are our default messages:
+          extensions {
+            commitStatus {
+              context('ci/jenkins/pr')
+              triggeredStatus("Triggering workflow-e2e/e2e-runner build...")
```

Comment:
could just interpolate `name` here: `"Triggering ${name} build"` -- or even just use the Jenkins var (should be same) `"Triggering ${JOB_NAME} build"`